pulsar-broker/pom.xml

```diff
@@ -425,6 +425,25 @@
 
   <build>
     <plugins>
+      <plugin>
+        <groupId>org.gaul</groupId>
+        <artifactId>modernizer-maven-plugin</artifactId>
+        <configuration>
+          <failOnViolations>false</failOnViolations>
```

Comment:
should be true after as we've fixed all the violation?

Reply:
@MarvinCai please check this comment https://github.com/apache/pulsar/pull/16993#discussion_r945935655

I'm OK since this patch does improve the situation.